### ISSUE TITLE
Draft events should be able to have more organizers added

### ISSUE THREAD

**Issue by lilliealbert** [railsbridge/bridge_troll#395]
Not sure how hard this will be, but in order for co-organizers to collaborate on an event, the person who originally creates it has to submit it for approval before any other organizers can edit it.

This creates the pretty annoying situation where users have to ask the admin-types *not* to approve an event so that they can collaborate on it. Which means emails and uncertainty, since more than one person can approve events.

**Comment by carlisia** [comment]
:+1:  for this.

**Comment by ultrasaurus** [comment]
:+1:

**Comment by carlisia** [comment]
I would like to circle back to this and add one more POV. The way the system is designed it assumes that an event is ready to be published once it has 1 organizer and date/location. This might work well for well established tech in big tech hubs (say, Rails in SF), because it is a build and they will come type of thing, in other words, have an event ready to go and not only students but also teachers and TAs will show up.

But for emerging tech or tech in places that are not big tech hubs this doesn't work. For me to approve an event for GoBridge, for example, I need to make sure that all is in place, including a teacher.

For this additional reason, it would be very helpful for teachers and TAs to add themselves to an event that is in DRAFT mode. The date will most likely be worked out only after the these items are in place.

**Comment by carlisia** [comment]
Would this be something I could contribute with code to help out?

**Comment by carlisia** [comment]
Also, for the record, it would be entirely useful if not only organizers, but all of the volunteers could add themselves to an event still in draft mode. This would be extremely helpful to an organizer who has to keep track of who's willing to volunteer and coordinate with those a date that works. It would then be very simple for the ones for whom the date didn't workout to drop out. In other words, it would a good mechanism for all involved to get potential volunteers who show up early (before the event can be published) involved and in the loop, and to keep track of them.

**Comment by lilliealbert** [comment]
It would probably be helpful for us to chat about the process that you're using that's driving these requests. It would be a pretty big departure from the way the current system works, and I need to be confident it would be the right change for all the users of Bridge Troll before we put that work in. Maybe we can sync up on Slack?

**Comment by lilliealbert** [comment]
@carlisia Would you mind creating a separate issue for the discussion about volunteer RSVPs? I'd like this issue to remain focused on adding organizers.

**Comment by aneyzberg** [comment]
@carlisisa I don't think that having volunteers sign up to workshops that are in draft mode is a the best idea. The reason that workshops in draft mode exist is so that they can be approved and ensure that all events posted on Bridgetroll are compliant with  the Bridge Foundry guidelines. So it seems like it would make sense to have an event approved before people are allowed to sign up for it.

**Comment by carlisia** [comment]
Hi @aneyzberg and @lilliealbert, let me use the system some more and see how much of a pain point this really is. I'm taking in your comments and trying to see things from that POV as well. So yes, let's keep this issue focused on adding additional organizers, and thanks.

**Comment by scottyschup** [comment]
Going back to the original issue, I think it would be nice if multiple organizers can be added in Draft mode. That way co-organizers can proof the details before the event is submitted for posting. Right now this can only be done through screenshots or copy/pastes.
